container_files/public_html/doc/builtin/procedures/Classifier.md

```diff
@@ -98,6 +98,34 @@ when a generic classification step is required.
 
 ![](%%jmlclassifier decision_tree)
 
+#### Parameter `update_alg`
+The parameter update_alg can take three different values: normal, gentle and prob.
+'normal' and 'gentle' are described in more details in this 
+[paper](http://projecteuclid.org/download/pdf_1/euclid.aos/1016218223)
```

Comment:
my only minor comment here is I'd vote adopting a pattern when we cite papers of having the full reference (like Author bla, bla, year, conference) plus a link if we have it. The ref could be in another section right before the glz, like "See also" with the ref or something. but in any case, more than just the link